src/main/java/Duke.java

```diff
@@ -1,10 +1,226 @@
+import duke.task.Task;
+import duke.task.Todo;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.command.Command;
+import duke.command.CommandType;
+import duke.exception.DukeException;
+import duke.exception.ExceptionType;
+
+import  java.util.Scanner;
+
 public class Duke {
+
+    public static final String BYE_MESSAGE = "\n\tSee you! Have a nice day!\n";
+    public static final String DEADLINE_IDENTIFIER = "/by";
+    public static final String EVENT_IDENTIFIER = "/on";
+
+    /**
+     * prints a line of 100 character length.
+     *
+     */
+    public static void printLine() {
+
+        for(int i = 0; i<100; i++) {
+            System.out.print("-");
+        }
+        System.out.println();
+
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        printWelcomeMessage();
+
+        Scanner in = new Scanner(System.in);
+        Command command = new Command(in.nextLine());
+        Task[] tasks = new Task[100];
```

Comment:
Maybe you can try looking into ArrayList from Java so that you won't have to declare a fixed size array. What happens if user enter more than 100 task?

Reply:
Yep! Will do that in the next push.